pkg/nsx/services/mediator/mediator.go

```diff
@@ -67,8 +67,12 @@ func (serviceMediator *ServiceMediator) GetAvailableSubnet(subnetSet *v1alpha1.S
 			return *nsxSubnet.Path, nil
 		}
 	}
+	var tags []model.Tag
+	for k, v := range subnetSet.Labels {
+		tags = append(tags, model.Tag{Scope: common.String(k), Tag: common.String(v)})
```

Comment:
This is a good way to map between labels and tags. But we need to make sure we always use vm/pod labels in SubnetSet. Will let @timdengyun to review.